jenkins-pipelines/performance/scylla-enterprise/scylla-enterprise-perf-regression-throughput-non-shard-aware-i4i-tablets.jenkinsfile

```diff
@@ -0,0 +1,15 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+perfRegressionParallelPipeline(
+    backend: "aws",
+    region: "us-east-1",
+    test_name: "performance_regression_test.PerformanceRegressionTest",
+    test_config: '''["test-cases/performance/perf-regression.100threads.30M-keys-i4i-enterprise.yaml","configurations/perf-loaders-non-shard-aware-config.yaml", "configurations/disable_speculative_retry.yaml"]''',
```

Comment:
so technically this config should be deleted from master/v15:
configurations/perf-loaders-non-shard-aware-config.yaml

since it's not complete, and need to point to different users / keys 

if we want to disable shared_aware it should be done vi scylla configuration

Reply:
in the end I'm alter this configuration to disable shard aware with configuration of scylla
and `use_prepared_loaders: true`